server/routes/viewModels/taskLists/dmTasks.js

```diff
@@ -35,7 +36,7 @@ module.exports = ({ decisions, tasks, stage }) => {
   }
 
   if (addressWithdrawn || curfewAddressRejected) {
-    return [
+    const t = [
```

Comment:
is this supposed to be `tasks`? (a list of tasks)

Reply:
Yes, but 'tasks' is already in scope.  Could be 'rejectedAddressTasks' I suppose. At the time I couldn't think of a suitable name and then forgot about it.  I'll fix in my next PR.